engine/src/main/java/com/datatorrent/stram/cli/ApexCli.java

```diff
@@ -2842,18 +2861,15 @@ public void execute(String[] args, ConsoleReader reader) throws Exception
       }
 
       if (commandLineInfo.args.length > 0) {
-        String filename = expandFileName(commandLineInfo.args[0], true);
-
         // see if the first argument is actually an app package
-        try {
-          AppPackage ap = new AppPackage(new File(filename));
-          ap.close();
+        try (AppPackage ap = newAppPackageInstance(new URI(commandLineInfo.args[0]), false)) {
```

Comment:
Why not expand file name before passing it to open like it was being done before?